Source/WebCore/platform/EventTrackingRegions.cpp

```diff
@@ -28,7 +28,46 @@
 
 namespace WebCore {
 
-TrackingType EventTrackingRegions::trackingTypeForPoint(const String& eventName, const IntPoint& point)
+ASCIILiteral EventTrackingRegions::eventName(Event eventName)
```

Comment:
Let's also change the variable name from `eventName` to `event`.

Reply:
Done.